CHANGELOG.md

```diff
@@ -1,30 +1,91 @@
-##2015-09-02 - Unsupported Release 0.8.1
-
-###Summary
+## 2016-01-14 - Supported Release 1.0.0
+### Summary
+
+First supported release!
+
+### Features
+
+### Bugfixes
+
+SORT AND CLEANUP
+(MODULES-2514) Update module dependencies
+(MODULES-2480) Remove Unused Variable
+(MODULES-2559) Fix ERB Template Bug
+(MODULES-2559) Update Helper Library
+(MODULES-2575) Remove LCM Refresh Mode checks
+(MODULES-2466) Document ProductCode for Package DSC Resource
+(MODULES-2178) Store MOF info in Puppet parameters
+(MODULES-2178) Support MOF EmbeddedInstances
+(MODULES-2178) Munge EmbeddedInstance types
+(MODULES-2178) Validate EmbeddedInstance prop names
+(MODULES-2178) Validate EmbeddedInstance prop types
+(MODULES-2178) Generate PowerShell for PSCredential
+(MODULES-2537) Fix Helper Library Documentation
+(MODULES-2537) Force UTF-8 for Helper Library
+(MODULES-2521) Confine to PowerShell version 5
+(MODULES-2521) Add Confine to all Types
+(MODULES-2485) Remove LCM Pull RefreshMode
+(MODULES-2615) Prevent empty password for PSCredential
+(MODULES-2615) Handle Empty Passwords for PSCredentials
+(MODULES-2270) Define provider after type
+(MODULES-2244) Evaluate Supported DSC Resources
+(MODULES-2692) Update vendored DSC resources
+(MODULES-2244) Don't build if WaitFor* resources
+(MODULES-2244) Remove WaitFor* resources
+(MODULES-2244) Rake exclusions case insensitive
+(MODULES-2244) Update rake task to remove WaitFor DSC Resources
+(MODULES-2759) Remove unused signed int validation
+(MODULES-2759) Generate types for signed integer arrays
+(MODULES-2759) Update validation helpers with all signed integers
+(MODULES-2520) clean up generated type docs
+(MODULES-2520) Read only parameters
+(MODULES-2519) Places for more DSC Information
+(MODULES-2562) Ensure uints don't allow negative
+(MODULES-2562) Handle uint[] / int[] typed params
+(MODULES-2520) generate types.md
+(MODULES-2519) Add new DSC Documents home page location
+(MODULES-2668) Document Windows 2003 LFN issues
+(MODULES-2589) Bump Puppet Agent Version
+(MODULES-2520) Add link to Types document
+(MODULES-2777) Build all DSC Types
+(MODULES-2520) Simplify type template / regenerate
+(MODULES-2520) Remove dscmeta_import_resource
+(MODULES-2777) Rebuild all DSC Types at 84a467c
+(MODULES-2837) Specify DSC Resource Module Path
+(MODULES-2641) Invoke-DscResource triggers reboot
+(MODULES-2799) Build custom types
+(MODULES-2512) Cannot ensure absent on Service DSC Resource
+(MODULES-2709) Reuse PowerShell instance for DSC
+(MODULES-2709) Disable Ruby 1.9.3 x86 shared PS
```

Comment:
Added a PR about this change to the README at https://github.com/puppetlabs/puppetlabs-dsc/pull/174